addons/website_crm_partner_assign/controllers/main.py

```diff
@@ -164,7 +164,7 @@ def portal_my_opportunity(self, opp, **kw):
                 'opportunity': opp,
                 'user_activity': opp.sudo().activity_ids.filtered(lambda activity: activity.user_id == request.env.user)[:1],
                 'stages': request.env['crm.stage'].search([('is_won', '!=', True)], order='sequence desc, name desc, id desc'),
-                'activity_types': request.env['mail.activity.type'].sudo().search([]),
+                'activity_types': request.env['mail.activity.type'].sudo().search(['|', ('res_model_id.model', '=', opp._name), ('res_model_id', '=', False)]),
```

Comment:
Could probably be considered as a bugfix :D don't you think we should make a PR against stable for this ?